cves/CVE-2013-0861.yml

```diff
@@ -278,8 +312,10 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: true
+  note: |
+    the channel counter (channel vector before) is being used in multiple files 
```

Comment:
The data being used in multiple subsystems doesn't necessarily mean IPC occurred. Can you find any specific instances of IPC being used?

Reply:
Good catch, thanks!